mitodl/open-discussions
# Removed front-end email suggestion

#### What are the relevant tickets?
Closes #932

#### What's this PR do?
What the title says

#### How should this be manually tested?
Use any email form (`/login`, `/signup`), and type in something that would normally prompt a suggestion (e.g.: `someone@m1t.edu`). There should be no suggestion and the form should have the same behavior otherwise
